src/containers/Review/assignment/ReviewSectionRow.tsx

```diff
@@ -87,13 +87,19 @@ const ReviewSectionRow: React.FC<ReviewSectionRowProps> = ({
   return (
     <div key={`section_${sectionCode}_assignment_${assignmentId}`}>
       {canRenderRow && (
-        <Grid className="flex-row" verticalAlign="middle">
-          <Grid.Row columns={4} className="reviewer-row">
+        <Grid stackable columns={4} verticalAlign="middle">
+          <Grid.Column>
             <ReviewSectionRowAssigned {...props} />
+          </Grid.Column>
+          <Grid.Column>
             <ReviewSectionRowLastActionDate {...props} />
+          </Grid.Column>
+          <Grid.Column>
             <ReviewSectionRowProgress {...props} />
+          </Grid.Column>
+          <Grid.Column>
             <ReviewSectionRowAction {...props} />
-          </Grid.Row>
+          </Grid.Column>
         </Grid>
```

Comment:
I thought I would try to use Semantic's Grid Stackable property if possible.  If it's not what we want, I can find another way.

Reply:
![Screenshot 2023-11-07 at 12 12 29 PM](https://github.com/msupply-foundation/conforma-web-app/assets/5456533/9d6f9489-be89-42f0-a442-a1fdb5f1eebe)
Yeah that's great, thanks. Only thing I'd suggest is for the rows within each block to be a bit closer together.

Reply:
Reduced spacing... we'll see if it's enough, or needs more...